vignettes/WQPDataHarmonization.Rmd

```diff
@@ -669,7 +669,7 @@ that we want to remove the "Quality Control Sample-Field Replicate" and
 field.
 
 ```{r}
-TADAProfileClean14 <- dplyr::filter(TADAProfileClean13, !(ActivityTypeCode %in% c("Quality Control Sample-Field Replicate", "Quality Control Sample-Field Blank", "Quality Control Sample-Lab Duplicate", "Quality Control Sample-Equipment Blank")))
+TADAProfileClean14 <- dplyr::filter(TADAProfileClean13, !(ActivityTypeCode %in% ActivityTypeCode[grepl("Quality",ActivityTypeCode)]))
```

Comment:
add line for users # this removes rows where any value in the ActivityTypeCode includes the string "quality" (not case sensitive?), These quality control samples are not included in the analysis.  Are we now including this set as an automated step in autoclean, or just including it here? If the latter, we will need to remember to do this step separately in the shiny app as well.

Reply:
It doesn't happen in autoclean. Would it be better to do it there?

Reply:
Let's wait for now. It may be better as a flag that can then be used to inform use of that data in the future.